From 74ab54fc7eebb7ddf378cb9efcbff956e091a672 Mon Sep 17 00:00:00 2001
From: Li Jun <jun.li@nxp.com>
Date: Wed, 13 Jan 2016 11:56:15 +0800
Subject: [PATCH 0851/5242] MLK-12248-2 ARM: imx6sl-evk: add tx-d-cal property
 for usbphy

commit  15288dc2edd477cf942d755aaa3d7ed85557ee6e from
https://source.codeaurora.org/external/imx/linux-imx.git

Set tx-d-cal to be 0x5 to improve usb signal quality.

Signed-off-by: Li Jun <jun.li@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6sl-evk.dts |    8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm/boot/dts/imx6sl-evk.dts b/arch/arm/boot/dts/imx6sl-evk.dts
index e249317..1cc98f8 100644
--- a/arch/arm/boot/dts/imx6sl-evk.dts
+++ b/arch/arm/boot/dts/imx6sl-evk.dts
@@ -904,6 +904,14 @@
 	status = "okay";
 };
 
+&usbphy1 {
+	tx-d-cal = <0x5>;
+};
+
+&usbphy2 {
+	tx-d-cal = <0x5>;
+};
+
 &usdhc1 {
 	pinctrl-names = "default", "state_100mhz", "state_200mhz";
 	pinctrl-0 = <&pinctrl_usdhc1>;
-- 
1.7.9.5

